Build.ps1

```diff
@@ -22,6 +22,10 @@ if ($null -eq (Get-Command dotnet -ErrorAction SilentlyContinue))
 
 $currentPath = Split-Path $MyInvocation.MyCommand.Path -Parent
 $outputPath = Join-Path -Path $currentPath -ChildPath "bin"
+if (-not (Test-Path $outputPath))
```

Comment:
This needs to be moved after the clean up code that begins on line 30. Also, add `-PathType Container` to the test. 

I'm not sure what conditions cause this, but something was creating a `bin` file in the project directory. This test for the path would pass and folder creation was skipped because the path does exist but is a file instead of a folder. Then all the `Copy-Item` tasks were throwing errors because they were targeting a nonexistent directory. *shrugs